### PR TITLE
chore: cleanup deprecated project usage

### PR DESCRIPTION
**Pull request by kevcodez**
- Replaces usage of deprecated project usage endpoint
- Removes a bunch of unused prefetch calls
- Removes some unused API routes

### CODE REVIEW

File: studio/data/api.d.ts [hunk 1]
```diff
@@ -52,6 +52,10 @@ export interface paths {
      */
     get: operations['ProjectsResourceWarningsController_getProjectsResourceWarnings']
   }
+  '/platform/tos/fly': {
```
Comment (kevcodez) [inline studio/data/api.d.ts:55]: Just regenerated this using the npm command

File: studio/data/analytics/project-log-requests-count-query.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+export async function getProjectLogRequestsCountStats(
+  { projectRef }: ProjectLogRequestsCountVariables,
+  signal?: AbortSignal
+) {
+  if (!projectRef) {
+    throw new Error('projectRef is required')
+  }
+
+  const response = await get<ProjectLogRequestsCountResponse>(
```
Comment (joshenlim) [inline studio/data/analytics/project-log-requests-count-query.ts:25]: let's use the new get method from data/fetchers
they're typed with the API specs

Comment (joshenlim) [reply studio/data/analytics/project-log-requests-count-query.ts:25]: also just to double check if this endpoint is already available on prod?

File: studio/components/interfaces/Settings/Database/DiskSizeConfiguration.tsx [hunk 3]
```diff
@@ -62,7 +60,7 @@ const DiskSizeConfiguration = ({ disabled = false }: DiskSizeConfigurationProps)
     updateProjectUsage({ projectRef, volumeSize })
   }
 
-  const currentDiskSize = projectUsage?.disk_volume_size_gb ?? 0
+  const currentDiskSize = project?.volumeSizeGb ?? 0
```
Comment (joshenlim) [inline studio/components/interfaces/Settings/Database/DiskSizeConfiguration.tsx:63]: is the volumeSizeGb change already updated on prod?

Comment (kevcodez) [reply studio/components/interfaces/Settings/Database/DiskSizeConfiguration.tsx:63]: This was already present before actually

File: studio/data/analytics/project-log-requests-count-query.ts (hunk 2 above)
Comment (kevcodez) [reply studio/data/analytics/project-log-requests-count-query.ts:25]: It's on prod yes, will use the new data fetcher

**Review by joshenlim — APPROVED** [review]

**Pull request merged**